test/test_xesmf.jl

```diff
@@ -0,0 +1,57 @@
+include("dependencies_for_runtests.jl")
+
+using XESMF
+using SparseArrays
+using LinearAlgebra
+
+function regrid_conservatively!(dst_field, weights, src_field)
+    @assert src_field.grid.Nz == dst_field.grid.Nz
+
+    for k in 1:src_field.grid.Nz
+        LinearAlgebra.mul!(vec(interior(dst_field, :, :, k)), weights, vec(interior(src_field, :, :, k)))
```

Comment:
for the test this might be ok, but as a reminder: it is 1000x more expensive to do this rather than having contiguous memory, for example
```julia
wrk .= vec(interior(src_field, :, :, k) # wrk is a previously allocated array
LinearAlgebra.mul!(wrk2, weights, wrk)
vec(interior(src_field, :, :, k)) .= wrk2 # wrk2 is a previously allocated array
```

Reply:
Gotcha! That's probably part of the slowness I'm observing.....! Thanks!!

Reply:
We need to build a `Regridder` object. Inside that, we can allocate scratch memory.